src/mesh/gmsh_utils.jl

```diff
@@ -417,7 +424,13 @@ function gen_rectangle_mesh(
     gmsh.model.mesh.partition(n_partitions)
 
     # Write result
+    rm(output; force = true)
     gmsh.write(output)
+    if write_geo
```

Comment:
wouldn't it be nicer to provide an array of output names? And then we would do `gmsh.write.(output)`